src/components/header/ControlOptionsList.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import ControlOption from "./ControlOption";
+import controlOptionsData from "./constants/controlOptionsData";
+import './ControlOptionsList.css'
+
+function ControlOptionsList(){
+    return (
+        <>
+            {controlOptionsData.map((item)=>{
+                return <ControlOption key={item.label} item={item} />
```

Comment:
Use the prop system to take the data and then render it and keep all major functionality outside jsx in a helper function